src/proc_image_processing/cpu/filters/custom/test_filter.h

```diff
@@ -13,35 +13,48 @@ namespace proc_image_processing {
     public:
         using Ptr = std::shared_ptr<TestFilter>;
 
-        explicit TestFilter(const GlobalParamHandler &globalParams)
+        explicit TestFilter(const GlobalParameterHandler &globalParams)
                 : Filter(globalParams),
                   x_("X", 0, -512, 512, &parameters_),
                   y_("Y", 0, -512, 512, &parameters_),
                   w_("Width", 200, 0, 1024, &parameters_),
                   h_("Height", 200, 0, 1024, &parameters_),
                   angle_("Angle", 0, 0, 360, &parameters_),
-                  header_("Header", "test", &parameters_),
-                  specField1_("SpecialField_1", "sf1", &parameters_),
-                  specField2_("SpecialField_2", "sf2", &parameters_) {
+                  header_("Header", "test_filter", &parameters_),
+                  specField1_("Special field 1", "sf1", &parameters_),
+                  specField2_("Special field 2", "sf2", &parameters_) {
             setName("TestFilter");
         }
 
         ~TestFilter() override = default;
 
-        virtual void init() {}
-
         void apply(cv::Mat &image) override {
-            target_.setTarget("test_filter", x_.getValue() - 1000 / 2, y_.getValue(),
-                              w_.getValue(), h_.getValue(), angle_.getValue(), 1000,
-                              -1000 - (1000 / 2), specField1_.getValue(),
-                              specField2_.getValue());
+            target_.setTarget(
+                    header_.getValue(),
+                    x_.getValue() - 1000 / 2,
+                    y_.getValue(),
+                    w_.getValue(),
+                    h_.getValue(),
+                    angle_.getValue(),
+                    1000,
+                    -1000 - (1000 / 2),
```

Comment:
huh ? maybe use constants to clarify the numbers.. or just put -1500.

Reply:
Dunno, was already there. Just reformated the stuff